dev/build-tool-support/src/main/scala/com/lightbend/lagom/dev/Reloader.scala

```diff
@@ -144,6 +156,11 @@ object Reloader {
         reloader.close()
       }
       def url(): String = server.mainAddress().getHostName + ":" + server.mainAddress().getPort
+      override def bindings(): Seq[DevServerBinding] =
+        Seq(
+          DevServerBinding("HTTP", httpAddress, httpPort),
+          DevServerBinding("HTTPS", httpAddress, httpsPort)
+        )
```

Comment:
`url` still implemented bu I think we could just drop it entirely (avoid the deprecation phase).

Then `bindings` is directly build from arguments instead of relying on `server.xyz()`. I used this approach to avoid having to make changes in play (see https://github.com/lagom/lagom/issues/1554#issuecomment-416328825).

Reply:
I was thinking about that as well. 

That's probably ok to drop it instead of deprecating it. Although this is not officially internal API, this is a dev mode tool and I don't see how this could direct used by a user. On the other hand, there is no harm in keeping it around.  

The same can be said about `LagomKeys.LagomServiceUrl`.